proto/booking/rpc/check_in_service.proto

```diff
@@ -18,7 +20,26 @@ message CheckInServiceRequestResponse {
 	.holms.types.booking.check_in.CheckInRequestResult result = 1;
 }
 
+message GetPreCheckinInfoResponse {
+	holms.types.booking.indicators.ReservationIndicator reservation = 1;
+	string vehicle_plates = 2;
+	string personal_identification = 3;
+	holms.types.booking.reservations.RecommendedCheckinAuthorizationAmounts recommended_authorizations = 4;
+}
+
+message SetPreCheckinInfoRequest {
```

Comment:
For request/response stuff defined inline like this with the service, I've been using the convention of `<ServiceName><MessageName>`, e.g. `CheckInServiceSetPreCheckinInfoRequest`. 

Do you like this convention? Dislike? Figured I'd ask since it's not being followed uniformly.